src/routes.js

```diff
@@ -66,7 +66,7 @@ const routes = [
     component: LessonCollection,
     options: { ssrData: ssrDataLoaders.lessonsCollectionPage }
   },
-  { path: ':routeType/:tab/cu/:id', component: PlaylistItemPage, options: { ssrData: ssrDataLoaders.cuPage } },
+  // { path: ':routeType/:tab/cu/:id', component: PlaylistItemPage, options: { ssrData: ssrDataLoaders.cuPage } },
```

Comment:
@wissog ?

Reply:
this causes Article Page to be displayed like a video item...
and i didn't find its contribution